lib/update/cluster/engine.go

```diff
@@ -316,8 +316,9 @@ func (f *engine) reconcilePlan(ctx context.Context) error {
 	}
 	f.plan = *plan
 	var buf bytes.Buffer
+	type s string
 	fsm.FormatOperationPlanText(&buf, f.plan)
-	f.Debugf("Reconciled plan: %v.", buf.String())
+	f.WithField("plan", s(buf.String())).Debug("Reconciled plan.")
```

Comment:
Why is the type conversion needed here?

Reply:
Ah, I've been meaning to comment that - will do. Basically, it is an ugly workaround to avoid plan output being escaped when going through the text log formatter.